src/hooks/use-sse.tsx

```diff
@@ -1,19 +1,59 @@
-import { useContext, useEffect } from "react";
+import { useCallback, useContext, useEffect } from "react";
 import { toast } from "react-toastify";
 import { AppStatus } from "../models/app-status";
 import { App } from "../models/app.model";
 import { BtcInfo } from "../models/btc-info";
 import { HardwareInfo } from "../models/hardware-info";
+import { InstallAppData } from "../models/install-app";
 import { LnStatus } from "../models/ln-status";
 import { SystemInfo } from "../models/system-info";
 import { WalletBalance } from "../models/wallet-balance";
 import { SSEContext, SSE_URL } from "../store/sse-context";
+import { availableApps } from "../util/availableApps";
 import { setWindowAlias } from "../util/util";
 
+/**
+ * Establishes a SSE connection if not available yet & attaches / removes event listeners
+ * to the single events to update the SSEContext
+ * @returns the infos from the SSEContext
+ */
 function useSSE() {
   const sseCtx = useContext(SSEContext);
   const { evtSource, setEvtSource } = sseCtx;
 
+  const appInstallSuccessHandler = useCallback(
+    (installData: InstallAppData, appName: string) => {
+      if (installData.mode === "on") {
+        if (
+          installData.httpsForced === "1" &&
+          installData.httpsSelfsigned === "1"
+        ) {
+          toast(
+            `Install finished :)\n\nYou may need to accept self-signed HTTPS certificate in your browser on first use.`
```

Comment:
i18n?

Reply:
Comes with #261 just wanted to iterate since the code is already in master.